samples/openapi3/client/petstore/python-pydantic-v2/tests/test_model.py

```diff
@@ -291,13 +291,14 @@ def test_anyOf(self):
             p2 = petstore_api.AnyOfPig.from_json("1")
             self.assertTrue(False)  # this line shouldn't execute
         except ValueError as e:
+            self.maxDiff = None
             error_message = (
                 "No match found when deserializing the JSON string into AnyOfPig with anyOf schemas: BasquePig, "
                 "DanishPig. Details: 1 validation error for BasquePig\n"
-                "__root__\n"
-                "  BasquePig expected dict not int (type=type_error), 1 validation error for DanishPig\n"
-                "__root__\n"
-                "  DanishPig expected dict not int (type=type_error)")
+                "  Input should be a valid dictionary or instance of BasquePig [type=model_type, input_value=1, input_type=int]\n"
+                "    For further information visit https://errors.pydantic.dev/2.3/v/model_type, 1 validation error for DanishPig\n"
```

Comment:
It is not wise to include the document url in the test
Since pydantic v2.4.0 was released a few hours ago, the URL has changed and the test will fail

Reply:
agreed. we didn't take into consideration upgrading pydantic to newer version when writing the test. please feel free to revise the test